vivarium/plots/plot.py

```diff
@@ -0,0 +1,446 @@
+"""
```

Comment:
Should these plotting functions go into separate files? That would let us group utility functions like `order_list_of_paths` with the functions that actually use them, and that might be a more scalable approach if we add plots in the future. Also, we don't really need to have a whole `plots` folder if it's only going to contain one file. Just an idea though; I don't have strong preferences on this.